src/styles/themeManager.js

```diff
@@ -0,0 +1,257 @@
+import merge from 'lodash.merge';
+import update from 'react-addons-update';
+import Colors from './colors';
+import ColorManipulator from '../utils/color-manipulator';
+import extend from '../utils/extend';
+import lightBaseTheme from './baseThemes/lightBaseTheme';
+import zIndex from './zIndex';
+
+/**
+ * get the MUI theme corresponding to a base theme.
+ */
+export function getMuiTheme(baseTheme) {
```

Comment:
Could we add a second parameter called `muiTheme`, that would override some value of the outputted muiTheme.
Right now, when you want to change some value of the `muiTheme`, you have to use, for instance
```js
muiTheme.appBar = Object.assign({}, muiTheme.appBar, {
  textColor: colors.white,
  height: 56,
});
```